bescort.lic

```diff
@@ -1474,6 +1489,30 @@ class Bescort
     end
   end
 
+  def ferry_textbook
+    start_script('first-aid')
```

Comment:
I'm a little worried about calling `first-aid` because it may try to move around. For example: https://github.com/rpherbig/dr-scripts/blob/master/first-aid.lic#L19

What do you think?

Reply:
I was unaware of the bleed bot setting. That's not something I've ever had enabled on this side. Suggestions?

Reply:
A few options I can think of offhand:
1. Return if `settings.bleed_bot`
2.  Add a 'nomove' mode to `first-aid` (optional script argument, I would think)
3. Add a 'no-bleedbot' mode to `first-aid` (similar but slightly different intent to option 2)